vaccine-standby/functions/setup.js

```diff
@@ -0,0 +1,507 @@
+            next: 'LanguagePreference',
+            event: 'incomingMessage',
+          },
+          {
+            event: 'timeout',
+          },
+          {
+            event: 'deliveryFailure',
+          },
+        ],
+        properties: {
+          offset: {
+            x: 300,
+            y: 2950,
+          },
+          service: '{{trigger.message.InstanceSid}}',
+          channel: '{{trigger.message.ChannelSid}}',
+          from: '{{flow.channel.address}}',
+          body: 'What is your email address?',
+          timeout: '3600',
+        },
+      },
+      {
+        name: 'Goodbye',
+        type: 'send-message',
+        transitions: [
+          {
+            event: 'sent',
+          },
+          {
+            event: 'failed',
+          },
+        ],
+        properties: {
+          offset: {
+            x: 60,
+            y: 3460,
+          },
+          service: '{{trigger.message.InstanceSid}}',
+          channel: '{{trigger.message.ChannelSid}}',
+          from: '{{flow.channel.address}}',
+          to: '{{contact.channel.address}}',
+          body: "Thank you, you are now now on the vaccine standby list.\n\nWe will notify you when you're eligible to receive a vaccine.",
+        },
+      },
+      {
+        name: 'LanguagePreference',
+        type: 'send-and-wait-for-reply',
+        transitions: [
+          {
+            next: 'Goodbye',
+            event: 'incomingMessage',
+          },
+          {
+            event: 'timeout',
+          },
+          {
+            event: 'deliveryFailure',
+          },
+        ],
+        properties: {
+          offset: {
+            x: 60,
+            y: 3210,
+          },
+          service: '{{trigger.message.InstanceSid}}',
+          channel: '{{trigger.message.ChannelSid}}',
+          from: '{{flow.channel.address}}',
+          body: 'What is your preferred language?\n1. English\n2. Español\n3. 繁體中文\n\nReply"1", "2", or "3"',
+          timeout: '3600',
+        },
+      },
+      {
+        name: 'ZipCode',
+        type: 'send-and-wait-for-reply',
+        transitions: [
+          {
+            next: 'EssentialWorker',
+            event: 'incomingMessage',
+          },
+          {
+            event: 'timeout',
+          },
+          {
+            event: 'deliveryFailure',
+          },
+        ],
+        properties: {
+          offset: {
+            x: 360,
+            y: 1120,
+          },
+          service: '{{trigger.message.InstanceSid}}',
+          channel: '{{trigger.message.ChannelSid}}',
+          from: '{{flow.channel.address}}',
+          body: 'What is your 5-digit zip code?',
+          timeout: '3600',
+        },
+      },
+    ],
+    initial_state: 'Trigger',
+    flags: {
+      allow_concurrent_calls: true,
+    },
+  };
+
+  const client = context.getTwilioClient();
+
+  // Deploy Twilio Studio Flow
+  function deployStudio(formattedFlow) {
+    return new Promise((resolve, reject) => {
```

Comment:
This is a double promise that isn't really necessary. You could do `return client.studio.flows.create(...).then(flow => flow.webhookUrl).catch(err => throw new Error(err.details))`

Reply:
Applies for the others as well

Reply:
+1

I just helped a developer on Stack Overflow who had unnecessarily wrapped these promises in another promise.

Reply:
Good call, removed the extra promise wrapper 👍